Xamarin.Forms.Core/ReflectionExtensions.cs

```diff
@@ -1,32 +1,38 @@
 ﻿using System;
 using System.Collections.Generic;
+using System.ComponentModel;
 using System.Linq;
 using System.Reflection;
 
-namespace Xamarin.Forms
+namespace Xamarin.Forms.Internals
 {
-	internal static class ReflectionExtensions
+	public static class ReflectionExtensions
```

Comment:
Again, in the internals namespace so it's not part of the supported public surface area. It's prob here so it can be shared across all the platforms. If we want to refactor to pull this into each platform or into a utils assembly we can do that in a separate PR.